src/Components/NewCommentForm.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+import { getAllUsers } from "../utils/getAllUsers";
+import { postComment } from "../utils/postComment";
+import { useParams } from "react-router-dom";
+
+const NewCommentForm = ({ setComments }) => {
+	const [isLoading, setIsLoading] = useState(true);
+	const [users, setUsers] = useState({});
+	const [newComment, setNewComment] = useState("");
+	const [selectedUser, setSelectedUser] = useState("");
+	const [submitted, setSubmitted] = useState(false);
+	const { article_id } = useParams();
+
+	useEffect(() => {
+		setIsLoading(true);
+		getAllUsers()
+			.then((data) => {
+				setUsers(data.users);
+			})
+			.then(() => {
+				setTimeout(() => {
+					setIsLoading(false);
```

Comment:
why are we setting loading to false two seconds after the users have successfully come back?